src/Python/dhart/graphgenerator/graph_generator.py

```diff
@@ -143,3 +143,13 @@ def GenerateGraph(
         return Graph(pointer)
     else:
         return None
+
+def CalculateAndStoreStepTypes(
+    g: Graph,
+    bvh: EmbreeBVH
+) -> None:
```

Comment:
There should be a docstring here, that includes examples, similar to the other methods in the file.